pkg/diff2/diff2.go

```diff
@@ -44,51 +44,72 @@ type Change struct {
 /*
 General instructions:
 
-  changes, err := diff2.ByRecord(existing, dc, nil)
-  //changes, err := diff2.ByRecordSet(existing, dc, nil)
-  //changes, err := diff2.ByLabel(existing, dc, nil)
-  if err != nil {
-    return nil, err
-  }
-
-  var corrections []*models.Correction
-
-  for _, change := range changes {
-    switch change.Type {
-    case diff2.REPORT:
-      corr = &models.Correction{Msg: change.MsgsJoined}
-    case diff2.CREATE:
-      corr = &models.Correction{
-        Msg: change.MsgsJoined,
-        F: func() error {
-          return c.createRecord(FILL_IN)
-        },
-      }
-    case diff2.CHANGE:
-      corr = &models.Correction{
-        Msg: change.MsgsJoined,
-        F: func() error {
-          return c.modifyRecord(FILL_IN)
-        },
-      }
-    case diff2.DELETE:
-      corr = &models.Correction{
-        Msg: change.MsgsJoined,
-        F: func() error {
-          return c.deleteRecord(FILL_IN)
-        },
-      }
-	  default:
-		  panic("unhandled change.TYPE %s", change.Type)
-    }
-
-    corrections = append(corrections, corr)
-  }
-
-  return corrections, nil
+	  changes, err := diff2.ByRecord(existing, dc, nil)
+	  //changes, err := diff2.ByRecordSet(existing, dc, nil)
```

Comment:
Delete these commented out code.

Reply:
You have them in your example as well ;)  I didn't add them. 
I re-copied the comment in your original file and made changes to them. Can you re-review them?